src/plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -674,6 +667,7 @@ def create_particles(
         self.particle_energy = particle_energy.to(u.eV).value
```

Comment:
There's a little bit of subtlety here because `_angles_monte_carlo` and `_angles_uniform` both read (and write in `_angles_uniform`) `self.nparticles`, which should in theory only be set by `load_particles`. Should I refactor these methods to take in an `nparticles` argument?

Reply:
Can you make that variable local and only set `self.nparticles` in `load_particles`? 

Or yeah, adding an nparticles argument might be the better way of doing it.